### DISCUSSION TITLE
Usage issues

### DISCUSSION THREAD

**Discussion by Memorablea** [community/community#59414] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/125779492/59152ffc-6d12-49d0-b854-72213d9711e3)
![image](https://github.com/community/community/assets/125779492/721ebed6-817c-44b1-a57e-2cfbcd5d8c9d)
May I ask how to set this option?

**Comment by Memorablea** [comment]
UP

**Comment by HarlanHaller** [comment]
Do you mean making the box checked like this:
- [x] your text
 
simply put an x in the box like this ` - [x] your text`

**Reply by HarlanHaller** [reply]
you can also check the box by clicking on it when editing the issue after it is created

**Reply by GhostOf0days** [reply]
Also, outside of that, you can use ```:heavy_check_mark:``` and ```:x:```, respectively for markdown.

**Reply by Memorablea** [reply]
No, what I mean is how to use this box?

**Reply by Memorablea** [reply]
```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```

You can see it here：https://github.com/sars-cov-2-variants/lineage-proposals/issues/new

**Reply by GhostOf0days** [reply]
As HarlanHaller said,
```
- [ ] Task 1
- [ ] Task 2
- [x] Task 3
```
A box with x represents putting a checkmark in that box. You can toggle the checkboxes by replacing [ ] with [x] for completed tasks or vice versa. There must be no spaces next to the x.

Here is how the above looks:
- [ ] Task 1
- [ ] Task 2
- [x] Task 3

**Reply by GhostOf0days** [reply]
This also allows you to manually toggle them by clicking on the box.

**Reply by Memorablea** [reply]
I know this, but my problem is not that

**Reply by GhostOf0days** [reply]
If you are referring to referencing an issue, use #. So #45 would bring up issue #45.

**Reply by GhostOf0days** [reply]
It would look like this: 
![image](https://github.com/community/community/assets/70134418/66a73597-f4f5-453d-a947-e43e9bc71377)

**Reply by Memorablea** [reply]
I am referring to how this box appears
![image](https://github.com/community/community/assets/125779492/71b6ec31-fa86-4550-877e-e2bce895254d)

**Reply by GhostOf0days** [reply]
```[tasklist]
### My tasks
- [ ] #45 
```

**Reply by Memorablea** [reply]
Okay, okay, forced to end this issue

**Reply by GhostOf0days** [reply]
tasklist won't show up, but add [tasklist] at the top of that.

**Reply by GhostOf0days** [reply, marked as answer]
Just for future reference, another option is to mark an answer by clicking the 3 dots on a comment. This would populate the answer right under your question.
